src/main/java/uk/gov/hmcts/reform/pip/data/management/service/publication/PublicationCreationRunner.java

```diff
@@ -42,7 +42,19 @@ public PublicationCreationRunner(PublicationService publicationService, Artefact
      * @return Returns the artefact that was created.
      */
     public Artefact run(Artefact artefact, String payload) {
```

Comment:
I'm wondering if this method is needed. Could the source callers call the other endpoint with true instead?

Reply:
Updated